addons/revolutionaries/CfgVehicles_O_Tanoan.hpp

```diff
@@ -145,6 +145,103 @@ class TACU_Revolutionaries_U_O_Tanoan_Autorifleman: TACU_Revolutionaries_U_O_Tan
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "V_Pocketed_black_F", "G_Balaclava_blk"};
+    backpack = "TACU_Revolutionaries_B_Rocketeer_Green";
+    weapons[] = {"launch_RPG32_green_F","CUP_hgun_Mac10", "Throw", "Put"};
+    respawnWeapons[] = {"launch_RPG32_green_F","CUP_hgun_Mac10", "Throw", "Put"};
+    magazines[] = {mag_4("CUP_30Rnd_45ACP_MAC10_M")};
+    respawnMagazines[] = {mag_4("CUP_30Rnd_45ACP_MAC10_M")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Tanoan_Saboteur: TACU_Revolutionaries_U_O_Tanoan_Rifleman01 {
+    displayName = "Saboteur";
+    uniformClass = "U_I_C_Soldier_Bandit_5_F";
+    Items[] = {mag_5("ACE_fieldDressing"), "ACE_Clacker"};
+    respawnItems[] = {mag_5("ACE_fieldDressing"), "ACE_Clacker"};
+    backpack = "TACU_Revolutionaries_B_Saboteur_Green";
+    weapons[] = {"hgun_PDW2000_F", "Throw", "Put"};
+    respawnWeapons[] = {"hgun_PDW2000_F", "Throw", "Put"};
+    magazines[] = {mag_6("30Rnd_9x21_Mag")};
+    respawnMagazines[] = {mag_6("30Rnd_9x21_Mag")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Tanoan_Pistoleer01: TACU_Revolutionaries_U_O_Tanoan_Rifleman01 {
+    displayName = "Pistoleer (Garch)";
+    uniformClass = "U_I_C_Soldier_Bandit_4_F";
+    linkedItems[] = {DEFAULT_ITEMS, "V_Pocketed_black_F", "G_Balaclava_blk"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "V_Pocketed_black_F", "G_Balaclava_blk"};
+    backpack = "";
+    weapons[] = {"hgun_Rook40_F", "Throw", "Put"};
+    respawnWeapons[] = {"hgun_Rook40_F", "Throw", "Put"};
+    magazines[] = {mag_6("16Rnd_9x21_Mag")};
+    respawnMagazines[] = {mag_6("16Rnd_9x21_Mag")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Tanoan_Pistoleer02: TACU_Revolutionaries_U_O_Tanoan_Rifleman01 {
+    displayName = "Pistoleer (FNX)";
+    uniformClass = "U_I_C_Soldier_Bandit_2_F";
+    backpack = "";
+    weapons[] = {"hgun_Pistol_heavy_01_F", "Throw", "Put"};
+    respawnWeapons[] = {"hgun_Pistol_heavy_01_F", "Throw", "Put"};
+    magazines[] = {mag_6("11Rnd_45ACP_Mag")};
+    respawnMagazines[] = {mag_6("11Rnd_45ACP_Mag")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_O_Tanoan_Head: TACU_Revolutionaries_U_O_Tanoan_Rifleman01 {
+    displayName = "Head";
+    uniformClass = "U_Marshal";
+    linkedItems[] = {DEFAULT_ITEMS, "V_LegStrapBag_black_F", "H_Hat_brown"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "V_LegStrapBag_black_F", "H_Hat_brown"};
+    backpack = "";
+    weapons[] = {"cup_hgun_Deagle", "Throw", "Put"};
+    respawnWeapons[] = {"cup_hgun_Deagle", "Throw", "Put"};
+    magazines[] = {mag_5("CUP_7Rnd_50AE_Deagle")};
+    respawnMagazines[] = {mag_5("CUP_7Rnd_50AE_Deagle")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
```

Comment:
```suggestion
    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Tanoan_Autorifleman.jpg);
    headgearList[] = {
        "H_Hat_brown", 1
    };
```

And CBA_facewearList if you don't want a balaclava